src/qaqc/01-organize_raw_data.Rmd

```diff
@@ -337,21 +350,19 @@ summarize_site_param <- function(site_arg, parameter_arg, api_data) {
     mutate(DT_join = as.character(DT_round),
            site = site_arg,
            parameter = parameter_arg) %>% 
-    full_join(filter(dplyr::select(field_notes, where, visit_comments, sensor_malfunction, DT_join, site)),
-                 by = c('DT_join', 'site'))
+    full_join(filter(dplyr::select(deployment_record, sonde_employed, visit_comments, sensor_malfunction, DT_join, site)),
+                 by = c('DT_join', 'site')) %>% 
+    # determine if sonde was in the field or not
+    fill(sonde_employed)
```

Comment:
Out of curiosity... how fast is this step now, compared to your old way?

Reply:
The littlest bit faster, maybe. When I timed it the whole chunk takes about ~1 min to run (59.55sec). I feel like it took longer before.